src/styles/getModalStyles.js

```diff
@@ -131,13 +134,23 @@ export default (type, windowDimensions) => {
             animationOut = 'slideOutDown';
     }
 
+    // The following declarations are meant only to facilitate the modal padding
+    // calculation in the modal file.
+    if (!modalContainerStyle.paddingTop) {
+        modalContainerStyle = {...modalContainerStyle, paddingTop: 0};
+    }
+    if (!modalContainerStyle.paddingBottom) {
+        modalContainerStyle = {...modalContainerStyle, paddingBottom: 0};
+    }
```

Comment:
Not a blocker, I think it might be better to remove this and just do `(modalContainerStyle.paddingTop || 0)` if we need a default value in the `Modal` file to do the padding calculations. It was a little unclear why we needed this until looking at the `Modal` file to see the addition.

Reply:
Ok! I wrote it like this because the ternary does get a little ugly with 4 `|| 0`, but it does make it clearer.

Reply:
Ok! I wrote it like this because the ternary does get a little ugly with 4 || 0, but it does make it clearer.
Will amend it.